eng/common/pipelines/templates/steps/convert-credScan-sarif-csv.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
```

Comment:
As we talked for now just put this template under the eng/pipelines/templates/steps and not under eng/common and then please file an issue in the tools repo to move this to eng/common and assign it to me.